platform/darwin/src/MGLAccountManager.m

```diff
@@ -93,4 +108,20 @@ + (NSURL *)apiBaseURL {
     return [MGLAccountManager sharedManager].apiBaseURL;
 }
 
++ (void)setSkuToken:(NSString *)skuToken {
+    NSTimeInterval oneHour = 60 * 60; // TODO: make this const
+    MGLAccountManager.sharedManager.skuTokenExpiration = [NSDate dateWithTimeIntervalSinceNow:oneHour];
+
+    MGLAccountManager.sharedManager.skuToken = skuToken;
+}
+
++ (NSString *)skuToken {
+    return [MGLAccountManager.sharedManager isSKUTokenExpired] ? [MGLSKUToken skuToken] : MGLAccountManager.sharedManager.skuToken;
+}
+
+- (BOOL)isSKUTokenExpired {
```

Comment:
Should this be a class method?

Reply:
I dunno, to be honest. This singleton only has class methods, but I’m not entirely sure that’s necessary. This particular method is only used internally, so I’ve declared it as an instance method, but that’s not a very considered decision.